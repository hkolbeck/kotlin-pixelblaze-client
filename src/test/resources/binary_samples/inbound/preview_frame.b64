BQAAAAEDAAoBAA4DAA8DAAUAAAkAAA0CAAkAAAoBAAsBABQJABEGAA8DAAQAAAQAAAQAAAQAAA0CAAsBAAkAAAECAAEDAAQAAAAAAAEBAAQAAAcAAAcAAAcAAAEDAAYAAAsBAAYAAAgAAAoBABYLABQJABIHAAcAAAgAAAoBAA0CABYLABQIABIGAAoBAAoBAAoBAAAAAAEDAAYAAAQAAAQAAAECAAUAAAkAABIGAA0CAAkAAAYAAAYAAAYAAAsBAAcAAAQAAAECAAAAAAECAAQAAA4DAAsBAAgAAAAAAAAAAAECAAAAAAECAAMAAAgAAAgAAAkAAAECAAQAAAcAAAEBAAUAAAsBABkPABMHAA4DAAoBAAoBAAsBAAsBABAFABcMABQIABEFAA8DAA==