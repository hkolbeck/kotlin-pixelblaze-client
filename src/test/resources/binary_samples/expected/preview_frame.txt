0
66304
655616
918272
983808
327680
589824
852480
589824
655616
721152
1313024
1115648
983808
262144
262144
262144
262144
852480
721152
589824
66048
66304
262144
0
65792
262144
458752
458752
458752
66304
393216
721152
393216
524288
655616
1444608
1313024
1181440
458752
524288
655616
852480
1444608
1312768
1181184
655616
655616
655616
0
66304
393216
262144
262144
66048
327680
589824
1181184
852480
589824
393216
393216
393216
721152
458752
262144
66048
0
66048
262144
918272
721152
524288
0
0
66048
0
66048
196608
524288
524288
589824
66048
262144
458752
65792
327680
721152
1642240
1246976
918272
655616
655616
721152
721152
1049856
1510400
1312768
1115392
983808
